Action action		# The action can cause any one of the effects
ActionEffect[] effects
